packages/tv/.prettierrc.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  bracketSpacing: false,
+  jsxBracketSameLine: true,
+  singleQuote: true,
+  trailingComma: 'all',
+};
```

Comment:
Can we remove this file? (Again, just for the sake of consistency)

Reply:
Removed